test/cpp/qps/qps_json_driver.cc

```diff
@@ -49,10 +49,103 @@ DEFINE_string(scenarios_file, "",
 DEFINE_string(scenarios_json, "",
               "JSON string containing an array of Scenario objects");
 DEFINE_bool(quit, false, "Quit the workers");
+DEFINE_bool(
```

Comment:
Let's go ahead and make this a string right away - defaulting to empty, and able to be set to "offered_load".

In the future I can envisage wanting to alter different values to find the load we want: number of channels, outstanding requests per channel, etc...

We don't have to implement multiple things yet, but since this parameter will start showing up in shell scripts, I'd like to be a little forward thinking with how we define it.